scanners/pageload.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from phantomas import Phantomas, PhantomasError
+
+command = Phantomas.CMD
+init = None
+
+def scan(domain, options):
+    logging.debug("[%s][pageload]" % domain)
+
+    if not (domain.startswith('http://') or
+            domain.startswith('https://')):
+        domain = 'http://' + domain
```

Comment:
Oddly enough (at least in the heady world of `.gov` domains) some domains don't bother setting a DNS entry for their root domain, or their redirect doesn't work correctly. One thing the `inspect` task does is determine the "canonical" endpoint for a domain, so we could have it look for the presence of a `cache/inspect/[domain].json` file and, if present, use it to determine what URL to construct.

We're doing something similar [in the tls scanner](https://github.com/18F/domain-scan/blob/master/scanners/tls.py#L24-L28) to not bother running SSL Labs scans on domains which we know don't have HTTPS.